packages/plugin-team/test/integration/spec/create.js

```diff
@@ -0,0 +1,183 @@
+/**!
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+import '../..';
+import '@ciscospark/plugin-wdm';
+
+import {assert} from '@ciscospark/test-helper-chai';
+import CiscoSpark from '@ciscospark/spark-core';
+import {find, findLast} from 'lodash';
+import testUsers from '@ciscospark/test-helper-test-users';
+import uuid from 'uuid';
+
+describe(`plugin-team`, () => {
+
+  let kirk, participants, spark, spock, team;
+
+  before(() => testUsers.create({count: 3})
```

Comment:
Because I need to test making teams with multiple users I need to make three users in this case.